scripts/config.py

```diff
@@ -4,3 +4,7 @@
 VESTING_SYMBOL = 'GLS'
 
 CYBERWAY_PREFIX='_CYBERWAY_'
+
+from utils import fixp_max
+_2s = 2 * 12000000000000
```

Comment:
Where does this strange constant come from? May be add some comment?

Reply:
Thank you for paying attention to this. It's came from hardcoded constant in database.cpp (get_content_constant_s()), and actually it's should be equal to 2 * 2000000000000. Fixed.